templates/global/table.html

```diff
@@ -21,7 +21,7 @@
       <td>{{ house.city.name }}</td>
       <td>{{ house.parent_profession_1 }}</td>
       <td>{{ house.parent_profession_2 }}</td>
-      <td>{{ house.income }}</td>
+      <td>{{ house.income }}$</td>
```

Comment:
Maybe you should add a space here so the dollar sign won't collide with the amount.

I remember we were talking about different currencies
I think we should keep it simple by addressing the amounts as dollars since it is a global currency like you did here
Do we all agree? @shlior7 @ShellyGolden @ronyyosef @Yoavsc129 @orIsrael

Reply:
in my opinion, dollars is ok

Reply:
was this resolved? @shlior7